src/v8runtime/V8Inspector.cpp

```diff
@@ -238,6 +240,31 @@ void InspectorClient::AwakePauseLockWithMessage(const std::string &message) {
   pauseWaitable_.notify_all();
 }
 
+void InspectorClient::DispatchProxy(const std::string &message) {
+  std::string normalizedString = stripMetroCachePrevention(message);
+
+  auto messageObj = folly::parseJson(message);
+  auto method = messageObj["method"].asString();
+
+  v8_inspector::StringView messageView(
+          reinterpret_cast<const uint8_t *>(normalizedString.data()),
+          normalizedString.size());
+
+  if (method == "Profiler.start" || method == "Profiler.stop") {
```

Comment:
maybe help me to add some comment of why we only run on js queue for these two commands.

Reply:
Those are the only one triggering v8::CpuProfiler that keeps some thread local stuff.